web/tailwind.config.ts

```diff
@@ -4,22 +4,31 @@ const config: Config = {
 	content: ["./src/pages/**/*.{js,ts,jsx,tsx,mdx}", "./src/components/**/*.{js,ts,jsx,tsx,mdx}", "./src/app/**/*.{js,ts,jsx,tsx,mdx}"],
 	theme: {
 		extend: {
-			backgroundImage: {
-				"gradient-radial": "radial-gradient(var(--tw-gradient-stops))",
-				"gradient-conic": "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
+			fontFamily: {
+				fira: ["var(--font-fira)"],
 			},
 			colors: {
-				primary: "var(--color-primary)",
-				secondary: "var(--color-secondary)",
-				tertiary: "var(--color-tertiary)",
-				"primary-text": "var(--color-primary-text)",
-				"secondary-text": "var(--color-secondary-text)",
-				"accent-text": "var(--color-primary)",
+				primary: "var(--c-primary)",
+				"primary-opacity": "rgba(69, 255, 2, 0.15)",
```

Comment:
Whats with that?

Reply:
That is slopiness on my side ;)
Added the defined css custom property with the hex value